radar/techniques.js

```diff
@@ -1,4 +1,33 @@
 const content = [
+  {
```

Comment:
Ik weet niet of ik het hier mee eens ben, vooral niet door wat er nu op "papier" staat. POM heeft nog steeds waarde, dit kan misschien per project en tool verschillen, maar het is er nog steeds. Daarom zou ik iets meer context hier willen zien omdat ik het idee heb dat hij nu maar met 1 context (tool) in het achterhoofd geschreven is